src/Example/Example.tsx

```diff
@@ -0,0 +1,73 @@
+import React, {useState, useEffect} from 'react';
```

Comment:
this should probably be a story only instead of what looks like a component in `src/`